src/main/resources/carpet.mixins.json

```diff
@@ -170,6 +170,7 @@
     "PointOfInterest_scarpetMixin",
     "ServerWorld_scarpetEventMixin",
```

Comment:
Curious, why just don't add to ServerWorld_scarpetEventMixin and potentially (its not events) change its name to ServerWorld_scarpetMixin?

Reply:
wait what?

Reply:
didnt notice lol

Reply:
literally 2 lines above, im blind